w06/test/gad/StudentResultForTest.java

```diff
@@ -0,0 +1,6 @@
+import gad.simplesort.Result;
+
+public class StudentResultForTest implements Result {
+    @Override
+    public void logArray(int[] array) {}
```

Comment:
This isn't the logging method of this exercise. Have a look in `StudentResult` which is given from Artemis.